src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/online-resource-dto.model.ts

```diff
@@ -0,0 +1,7 @@
+export class OnlineResourceDTO {
+    url: string;
+    title: string;
+    description: string;
+
+    constructor() {}
+}
```

Comment:
Is there any possibility that we can avoid this dto - as as far as i know we want to avoid unnecessary dtos as they require additional effort.

Reply:
Any suggestions? To me this was the simplest solution to transmit the data and have correct serialization/deserialization.

Reply:
I am also a fan of DTOs (but Stephan told me last time that we should avoid them :thinking:) - but i think you can leave it and if Stephan has another opinion about it, you can remove it afterwards too :+1: